HISTORY.md

```diff
@@ -1,3 +1,6 @@
+## 4.0.1 - 2022-03-07
+* Update to commonmarker 0.23.4; See [Version Changelog](https://github.com/gjtorikian/commonmarker/blob/main/CHANGELOG.md#v0234-2022-03-03)
```

Comment:
```suggestion
* Update to commonmarker 0.18.3; There isn't a version on RubyGems for this, so this is pointing to a [tag version on GitHub](https://github.com/gjtorikian/commonmarker/blob/v0.18.3/commonmarker.gemspec)
```

Reply:
@aharpole it might be helpful to add the context of the security patch that is in this version.